### PR TITLE
chore(*) goimports support

### PR DESCRIPTION
**Pull request by lobkovilya**
### Summary

Provide support of `goimports` util that allows to easily organize imports

### CODE REVIEW

File: Makefile.dev.mk [hunk 1]
```diff
@@ -128,3 +129,6 @@ dev/install/minikube: ## Bootstrap: Install Minikube
 
 dev/install/golangci-lint: ## Bootstrap: Install golangci-lint
 	curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(GOLANGCI_LINT_DIR) $(GOLANGCI_LINT_VERSION)
+
+dev/install/goimports: ## Bootstrap: Install goimports
```
Comment (jakubdyszkiewicz) [inline Makefile.dev.mk]: Let's add this target to `dev/tools/all`

File: Makefile [hunk 2]
```diff
@@ -284,7 +284,11 @@ vet: ## Dev: Run go vet
 golangci-lint: ## Dev: Runs golangci-lint linter
 	$(GOLANGCI_LINT_DIR)/golangci-lint run -v
 
-check: generate fmt vet docs golangci-lint ## Dev: Run code checks (go fmt, go vet, ...)
+.PHONY: imports ## Dev: Runs goimports in order to organize imports
```
Comment (jakubdyszkiewicz) [inline Makefile]: Can we add it to the top section of `.PHONY` instead?

File: app/kuma-dp/cmd/run_test.go [hunk 3]
```diff
@@ -15,13 +15,14 @@ import (
```
Comment (jakubdyszkiewicz) [inline app/kuma-dp/cmd/run_test.go]: Ok so I don't really get ordering here.
Why `"github.com/Kong/kuma/pkg/catalog"` is not merged with section of `github.com/Kong/kuma/app/kuma-dp/pkg/dataplane/envoy`? Does it respect manual separated sections or what is the reason? 🤔

Comment (lobkovilya) [reply app/kuma-dp/cmd/run_test.go]: Yes, looks like it tries to respect separation that already done. I didn't find any proves in documentation. But I deleted all "\n", run again and separation became correct.

File: Makefile.dev.mk (hunk 1 above)
Comment (lobkovilya) [reply Makefile.dev.mk]: It has already been added

Comment (jakubdyszkiewicz) [reply Makefile.dev.mk]: Sorry, missed that

**Pull request merged**